inc/remove_updates/namespace.php

```diff
@@ -8,6 +8,7 @@
 function bootstrap() {
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_maybe_update_checks', 1 );
 	add_action( 'admin_init', __NAMESPACE__ . '\\remove_update_nag' );
+	add_action( 'admin_init', __NAMESPACE__ . '\\remove_auto_updates' );
```

Comment:
Last thing to do here - I don't think `admin_init` will always be fired in the context of upgrades / installs so you can call the function directly from here, it's fine as we're just defining a constant so earlier the better.